database/migrations/2023_11_03_204729_create_books_table.php

```diff
@@ -41,6 +42,13 @@ public function up(): void
             $table->timestamps();
         });
 
+        Schema::create('publishers', function (Blueprint $table) {
```

Comment:
Modificar *migrations* não é indicado porque força os desenvolvedores a desfazer as *migrations* e executar elas novamente. O certo é criar uma nova contendo as modificações necessárias.